tests/test_utils.py

```diff
@@ -255,6 +255,25 @@ def test_load_schema_org(self):
 
 
 class TestDfUtils:
+    @pytest.mark.parametrize("preserve_raw_input", [True, False], ids=["Do not infer datatypes", "Infer datatypes"])
+    def test_load_df(self, helpers, preserve_raw_input):
+        test_col = "Check NA"
+        file_path = helpers.get_data_path("mock_manifests", "Invalid_Test_Manifest.csv")
+
+        df = df_utils.load_df(file_path, preserve_raw_input=preserve_raw_input, data_model=False)
+
+        assert df["Component"].dtype == "object"
+        assert df.shape[0] == 3
```

Comment:
is this part of the test trying to make sure that the empty rows have been removed? If so, could you make a comment in the code and make it clearer? I would suggest doing something like: 
```
original_df = pd.read_csv(file_path)
assert original_df.shape[0] == 4
df = xxx (call the load_df function) 
assert df.shape[0] == 3
```
`data_model=True` parameter doesn't seem to be covered by test. If this is also a parameter that we use in the code, could you add test for it too?